docs/registry/create/_index.md

```diff
@@ -1319,6 +1323,96 @@ Follow [these instructions](/cli/#using-the-build-subcommand) to automatically b
 
 {{% /alert %}}
 
+{{% /tab %}}
+{{% tab name="Python: venv" %}}
+
+Create a `run.sh` shell script that creates a new Python virtual environment, ensures that the package dependencies your module requires are installed, and runs your module.
```

Comment:
I think we should align the file names and code structure with https://github.com/viam-labs/python-example-module. We link to this example on the https://docs.viam.com/registry/upload/ page. So would likely be confusing to see both variations.

Reply:
Makes sense to me. We have a bunch of `run.sh` mentions in the docs, but no `exec.sh` yet. I've created https://viam.atlassian.net/browse/DOCS-2448 to handle that consideration separately. Let's leave this PR consistent with the rest of our docs for now.

Reply:
Ok sounds good. separate ticket is fine.